### DISCUSSION TITLE
Dictionary

### DISCUSSION THREAD

**Discussion by Boskizzi** [eworkers/K2ForJ4#53] [General]
Hello, 
I had a version of K2 in Italian. I have now installed K2forJ4 and I am experiencing a problem: instead of the various entries, in Italian, I see strings.

For example, instead of **Title** I see **K2_TITLE**, of **Featured** I see **K2_FEATURED**, etc.

How can I solve this? Thank you.
Boskizzi

**Comment by eworkers** [comment]
please share your Italian language files.

**Comment by Boskizzi** [comment]
Sorry, I made a big mistake!! 

In the backend I do not have the Italian language, but the English language.  But the problem is the same: I install k2forj4 and see the strings.

Sorry again,
Boskizzi
